src/realm/sync/protocol.hpp

```diff
@@ -315,6 +315,7 @@ enum class ProtocolError {
     compensating_write           = 231, // Client attempted a write that is disallowed by permissions, or modifies and
                                         // object outside the current query, and the server undid the modification
                                         // (UPLOAD)
+    bad_progress                 = 232, // Bad progress information (ERROR)
```

Comment:
This number has been updated to 233:
https://github.com/10gen/baas/blob/f0c1ad390a7f0ebbed8610766e3f0ca00d3133b2/realm/sync/protocol/protocol_errors.go#L167

Reply:
I will need to bump the baas version as well